populus/compilation/backends/solc_standard_json.py

```diff
@@ -114,10 +114,15 @@ def get_compiled_contracts(self, source_file_paths, import_remappings):
                 'remappings': import_remappings
             }
         }
-        std_input['settings'].update(self.compiler_settings)
 
+        std_input['settings'].update(self.compiler_settings.get("solidity_input_description", {}))
```

Comment:
If you can do this while you clean up the other things, mind splitting this across two lines.

```python
std_input_settings = self.compiler_settings.get("solidity_input_description", {})
std_input['settings'].update(std_input_settings)
```

slightly better readability.